src/VSPackage.cs

```diff
@@ -92,9 +93,14 @@ private async Task<bool> IsSolutionLoadedAsync()
             return value is bool isSolOpen && isSolOpen;
         }
 
-        private async void HandleOpenSolution(object sender = null, EventArgs e = null)
+        private void HandleOpenSolution(object sender, EventArgs e)
         {
-            await this.JoinableTaskFactory.SwitchToMainThreadAsync();
+            JoinableTaskFactory.RunAsync(HandleOpenSolutionAsync).Task.FileAndForget("StringResourceVisualizer");
```

Comment:
Use JoinableTaskFactory.Run instead of JoinableTaskFactory.RunAsync - it's designed for bridging async -> sync.